### ISSUE TITLE
Is this  MM-control project abandoned?

### ISSUE THREAD

**Issue by JohnnyTheOne** [prusa3d/MM-control-01#183]
Last development activity was 2 years ago.

**Comment by natthapolvanasrivilai** [comment]
Might be a 'yes'

**Comment by ssill2** [comment]
I was going to ask the same question.

**Comment by Cx01N** [comment]
The stability seems to be pretty poor recently and I haven't seen any development in a while. Might have to look at some alternatives.

**Comment by ssill2** [comment]
I know in episode #22 of the prusa live youtube show towards the end Josef mentioned an update coming to the mmu firmware.  That was about a month ago.  Maybe they've been working out of a different repo?  Multiple years between commits is not encouraging for sure.

**Comment by 5jvm0u4** [comment]
I've talked to their customer support and he told me "Not sure if there was any need to improve the current MMU FW version."

**Comment by ssill2** [comment]
So that means yes lol.  We all know the current firmware has issues lol.
Too bad

**Comment by TheDuggem** [comment]
https://github.com/prusa3d/Prusa-Firmware/issues/1876#issuecomment-1014292539

**Comment by 5jvm0u4** [comment]
> [prusa3d/Prusa-Firmware#1876 (comment)](https://github.com/prusa3d/Prusa-Firmware/issues/1876#issuecomment-1014292539)

I've saw this one as well, but that reply I got is from last week.

**Comment by 5jvm0u4** [comment]
![image](https://user-images.githubusercontent.com/75752327/160428809-706a235b-707e-4463-867b-b90e39362091.png)

**Comment by natthapolvanasrivilai** [comment]
I stop using my MMU because the hardware+software are full of bugs.
The 3D printed parts tolerance can be loosen. The hardware homing can be improved (as seen by TheZeroBeast repo).
The filament feed/retract/eject sequences can be improved and be less prone to minor error.

**Comment by Pyromane** [comment]
@leptun @DRracer @michalxfanta
Any Update?

**Comment by DRracer** [comment]
@Pyromane MM-control-01 is no longer developed/maintained. 
However, we keep on improving the MMU2S. In the near future, the firmware will be updated. Same goes for the MK3S+ and you can watch the development here https://github.com/prusa3d/Prusa-Firmware/tree/MK3_MMU2.

**Comment by ggppjj** [comment]
@DRracer Would it be worth marking this repo as "archived" and leaving that note in the readme? Good to hear, by the way!

**Comment by DRracer** [comment]
@ggppjj probably once the new MK3S+MMU FW combo goes out.

**Comment by ggppjj** [comment]
@DRracer apologies for the @, looks like that branch was pruned (or at least I'm getting a 404 on that link), is there a better way to keep an eye on progress?

**Comment by DRracer** [comment]
the new MMU support has been merged into the main MK3 branch meanwhile: https://github.com/prusa3d/Prusa-Firmware

**Comment by cjbaar** [comment]
@DRracer I see a lot of protocol changes in the 3.12 printer firmware related to the MMU, but I don't see anywhere that the actual MMU firmware is being developed publicly to match. Are you able to share this location with us?

**Comment by leptun** [comment]
The new mmu firmware is not public yet unfortunately.

**Comment by cjbaar** [comment]
That is unfortunate, and seemingly un-open-source-y. :) Is there a way to get beta access?

**Comment by ggppjj** [comment]
Likely there will be once the beta firmware is ready for testing 😁
